error-monitoring/test/blame_finder.test.ts

```diff
@@ -101,11 +101,11 @@ describe('BlameFinder', () => {
         })
         .reply(200, getGraphQLResponse('not-a-ref', path))
         .post('/graphql', ({query}) => {
-          expect(query).toContain(`ref(qualifiedName: "master")`);
+          expect(query).toContain(`ref(qualifiedName: "${mainBranch}")`);
```

Comment:
For tests, I think the expected value should be hardcoded. Changing the branch the app works on will require changing a few test expectations, and I think that's okay.

Reply:
```suggestion
          expect(query).toContain(`ref(qualifiedName: "master")`);
```

Reply:
Reverted.